src/utils.ts

```diff
@@ -102,24 +108,27 @@ const humanizeArray = (array, conjunction = 'and') => {
   return `${init.join(', ')}, ${conjunction} ${last}`;
 };
 
-const decryptReqs = (reqs, openPar = false, closePar = false) => {
-  const last = (i) => i === reqs.length - 2;
+const decryptReqs = (
+  reqs: PrerequisiteClause,
+  openPar: boolean = false,
+  closePar: boolean = false
+): string => {
+  const last = (i: number) => Array.isArray(reqs) && i === reqs.length - 2;
   let string = '';
 
-  if (!reqs[0])
+  if (!Array.isArray(reqs)) {
     string += (openPar ? '(' : '') + reqs.id + (closePar ? ')' : '');
-  else if (reqs[0] === 'and')
-    reqs.slice(1, reqs.length).forEach((req, i) => {
+  } else if (reqs[0] === 'and') {
+    const [, ...subClauses] = reqs;
+    subClauses.forEach((req, i) => {
       string += decryptReqs(req, i === 0, last(i)) + (last(i) ? '' : ' and ');
     });
-  else if (reqs[0] === 'or')
-    reqs.slice(1, reqs.length).forEach((req, i) => {
+  } else if (reqs[0] === 'or') {
+    const [, ...subClauses] = reqs;
+    subClauses.forEach((req, i) => {
       string += decryptReqs(req) + (last(i) ? '' : ' or ');
     });
-  else
-    reqs.forEach((req, i) => {
-      string += req.id + (i === reqs.length - 1 ? '' : ' or ');
-    });
```

Comment:
I'm not sure why this branch was there in the original code... it shouldn't be possible to hit from what I can tell.